plugins/aws/RunCbt.java

```
@@ -0,0 +1,85 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+
+import java.lang.reflect.Field;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.*;
+
+import com.amazonaws.services.lambda.runtime.Context;
+
+public class RunCbt {
+    public static void main(String[] args) {
+        return;
+    }
+
+    @SuppressWarnings("unchecked")
+    public String cbtHandler() {
+        String args = "compile";
+        File dir = new File("."); //to test locally change this to "target"
+        File[] filesList = dir.listFiles();
+        for (File file : filesList) {
+            File src = file;
+            File dest = new File("/tmp/" + file.getName());
+            try {    
+                copy(src, dest);
+            } catch (Exception e) {
+                e.printStackTrace();
+                return "Copy failed.";
```

swallowing an exception and simply printing a string instead hides errors. I would just not catch the exception and let it bubble up to the user instead. Gives more info to help diagnose the issue to the user :).